src/triage/component/results_schema/alembic/versions/5dd2ba8222b1_add_run_type.py

```diff
@@ -0,0 +1,54 @@
+"""add run_type
+
+Revision ID: 5dd2ba8222b1
+Revises: 079a74c15e8b
+Create Date: 2021-07-22 23:53:04.043651
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '5dd2ba8222b1'
+down_revision = '079a74c15e8b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('experiment_runs', sa.Column('run_type', sa.Text(), nullable=True), schema='triage_metadata')
+
+    op.add_column('experiment_runs', sa.Column('run_hash', sa.Text(), nullable=True), schema='triage_metadata')
+    op.drop_column('experiment_runs', 'experiment_hash', schema='triage_metadata')
```

Comment:
Here should we just rename the `experiment_hash` column to `run_hash` to avoid losing any existing data in the experiment hash column during the migration? Likewise, would be good to set the `run_type` for any existing rows as well via an `UPDATE` statement.